config/locales/en.yml

```diff
@@ -34,59 +34,59 @@ en:
     facet_disclosure:
       more: "Show more search options"
       fewer: "Show fewer search options"
-  checklists:
+  brexit_checker:
     title: "Get ready for Brexit: check what you need to do"
-  checklists_mailer:
+    results:
```

Comment:
Not sure we need this intermediate key? Not all the children key are specific to the results page - i.e. the breadcrumbs.

Reply:
I think the new key is better in that it's more consistent with the convention of having a separate namespace for each view (vs. `_results`). That said, I agree some of the keys within this namespace seem to be misplaced, so I'll propose a card to audit our locales.